builder/default.nix

```diff
@@ -71,8 +71,6 @@ let
             go run ${./symlink.go}
             ${lib.concatStringsSep "\n" localReplaceCommands}
 
-            find vendor
```

Comment:
I think it output too much noice.

Reply:
I agree, it was a temporary debugging thing that was left there by accident.